### ISSUE TITLE
Raise validation error for encrypted zip files

### ISSUE THREAD

**Issue by diox** [mozilla/addons-server#19994]
STR:
- Upload an encrypted zip file as your add-on file

Expected results:
- Specific error message explaining that the zip file shouldn't be encrypted

Actual results:
- Sorry, we couldn't load your webextension generic error

**Comment by diox** [comment]
@wagnerand : should we also prevent such uploads for source files ? Technically the developer might have provided the password to the reviewer in the notes.

**Comment by wagnerand** [comment]
> @wagnerand : should we also prevent such uploads for source files ? Technically the developer might have provided the password to the reviewer in the notes.

Yes. We want to make sure source files are accessible to admins at all times.

**Comment by diox** [comment]
I haven't checked but it's likely impossible to upload encrypted zips as source files at the moment, because we call `testzip()` when checking it, which would raise the same `RuntimeError` exception.

It's a little annoying that Python doesn't have a specific exception for this though... Not a big fan of having to catch `RuntimeError` ...

**Comment by KevinMind** [comment]
Old Jira Ticket: https://mozilla-hub.atlassian.net/browse/ADDSRV-132
